plugin.video.rivedila7/addon.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.rivedila7" name="La7-La7d" version="4.2.0" provider-name="Luivit, Testato">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0"/>
+    <import addon="script.module.requests"/>
+    <import addon="script.module.inputstreamhelper"/>
+    <import addon="script.module.beautifulsoup4" version="4.5.3"/>
+    <import addon="script.module.html5lib" version="0.999.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="addon.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">LA7-LA7d</summary>
+    <description lang="en_GB">Live Streaming, Replay the last week, On-Demand programs, News and Weather, Teche LA7 - The Protagonists</description>
+    <summary lang="it_IT">LA7-LA7d</summary>
+    <description lang="it_IT">Diretta Live, Rivedi ultima settimana, Programmi On-Demand, Telegiornali e Meteo, Teche LA7 - I Protagonisti</description>
+    <language>it</language>
+    <platform>all</platform>
+    <license>GNU General Public License v3.0</license>
```

Comment:
Please use the SPDX license identifier. A list can be found here: https://spdx.org/licenses/